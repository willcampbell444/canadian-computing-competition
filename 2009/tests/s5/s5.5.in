30000
1000
1000
520 4997 14558 408
993 3488 17655 140
134 14625 19624 608
882 23466 25693 105
399 10504 102 109
154 13672 18687 573
678 28082 3411 801
166 26491 4239 959
310 26037 10426 618
783 22302 26216 624
361 18916 11872 157
577 14595 21901 669
633 26975 9821 560
810 24786 9694 958
957 23487 20783 699
76 28474 3755 259
892 24737 22027 617
94 15674 24599 837
285 13806 10165 117
284 12861 10466 673
707 26916 16324 850
335 21868 21768 284
730 14291 618 620
500 13158 7002 384
363 3391 28522 826
600 3808 19664 484
28 8236 23016 104
214 8311 3252 265
170 11272 5346 286
551 1504 14885 680
528 7633 16145 945
240 22379 26222 562
319 27954 20635 935
539 14270 121 884
441 9918 399 325
291 23006 13507 605
438 15812 8479 807
612 16340 20794 968
74 21492 18391 612
500 4665 6496 813
526 8818 18963 791
454 11065 13993 278
601 895 10966 144
720 10891 28889 962
296 7509 1483 129
179 3259 18379 368
989 9604 7666 324
962 488 15019 672
614 14487 21413 237
391 4068 23535 195
150 7991 25854 237
741 19222 12571 513
301 4388 28846 134
158 24831 23833 682
719 14498 16574 748
995 1032 3705 906
439 4960 15002 193
9 18181 27721 773
905 20511 21620 392
40 17997 29499 855
447 14692 28794 613
130 2604 11848 483
418 15201 24598 961
781 5412 6063 437
875 26571 15956 350
442 1235 13586 950
733 698 11905 926
641 23772 9449 428
164 19439 21876 430
350 15294 4537 738
282 8119 4100 894
179 22062 7407 977
355 3053 19986 354
713 1148 6926 235
211 7445 27110 684
24 7203 9271 343
720 27188 10822 831
527 14069 4119 782
754 10808 26766 278
675 26284 7973 770
939 21029 20101 653
358 23651 11982 475
967 28920 27804 825
693 13343 7466 663
797 17765 15782 463
743 8323 13563 114
793 26496 11461 472
907 7467 17841 903
441 22846 9728 967
100 27150 10622 776
957 26419 6244 395
306 10001 10699 844
355 15454 6136 942
321 16449 29312 950
313 25114 6527 511
894 9220 1750 775
506 1686 6184 679
558 9958 15361 764
800 10514 2168 457
481 27457 15846 388
932 4187 18483 183
649 19605 9551 539
357 8961 13493 430
132 15250 22234 153
212 19989 25221 648
378 5769 3558 922
512 9529 20850 173
835 13344 26203 153
934 10119 22991 196
798 17934 21391 816
496 29154 5678 844
77 29979 17009 278
163 17640 22283 589
290 29540 15806 265
599 3153 13887 295
301 21433 11559 554
246 21586 9640 878
120 2043 22092 577
893 20968 6167 916
990 322 24278 802
882 12152 3226 865
115 5171 28738 121
869 11065 3451 264
901 8169 26252 123
729 25498 2331 952
739 18200 27888 724
740 27984 8198 139
81 22081 27480 265
338 19962 24646 217
443 26804 2660 828
681 4311 11717 959
821 17933 27596 318
911 16901 12858 136
111 21649 9143 373
435 14850 22008 919
257 9868 2594 140
494 3594 21681 515
823 936 6745 265
334 23855 12602 917
363 11155 7783 528
594 12273 731 821
478 6704 20673 907
248 16264 16146 957
702 6504 19344 136
976 21195 13681 912
462 10150 20484 446
154 9148 20853 527
416 29516 3201 695
567 14361 15470 260
20 23396 24780 612
156 24268 9445 769
453 28857 18877 490
417 780 15702 520
258 18230 2390 632
131 26411 24785 436
442 8898 9415 782
829 18360 25615 417
994 29848 26071 373
554 16149 22570 213
809 18006 22154 138
292 5225 10689 760
53 4901 26456 517
564 12535 27255 583
358 5357 26950 141
361 25538 6192 689
235 27706 16228 470
441 6788 20787 152
323 28249 13795 564
51 11966 4697 882
812 25103 27471 978
470 10728 5544 630
108 4827 20019 668
621 1820 23123 600
304 24855 27817 453
295 14096 63 632
773 617 29899 240
797 26823 8564 668
330 8609 23248 201
332 4799 14948 913
570 1439 28559 419
497 13191 21949 893
867 11800 17632 836
530 161 14829 539
109 13303 13637 890
863 14906 7189 963
314 8544 10466 102
619 1998 12536 701
716 22541 20313 672
533 12213 870 846
518 27399 4377 727
552 14047 19625 311
806 6661 4899 692
579 21669 11784 126
631 19244 22014 531
929 11601 29524 806
74 6645 25126 177
702 12607 21548 423
534 25219 1771 772
11 28086 3632 568
143 3816 29915 166
445 1721 1074 115
305 17427 20089 348
585 6586 21043 957
94 3658 18955 320
361 21147 14993 738
592 14895 10477 673
299 19602 22352 582
91 24441 16399 777
652 9887 9529 153
384 8308 28844 528
562 7968 20804 143
911 27008 3684 974
565 29624 8001 493
204 25156 17906 171
925 16854 16083 649
778 7367 1612 226
985 19781 7854 458
311 8369 14243 669
806 1225 10058 481
258 6717 8112 565
311 10822 7278 989
57 5514 9057 309
963 5333 13462 186
521 4092 19974 564
275 26506 23621 207
292 28937 18353 512
930 11449 27203 113
185 12187 8307 696
416 2847 27959 108
775 11824 8387 937
795 26737 11764 931
212 6315 22881 906
363 17838 3569 252
14 17006 10722 449
735 27943 28451 939
179 27271 10516 571
177 4594 6211 288
612 6303 6913 829
483 4406 22948 425
623 24046 4101 719
696 16985 17527 896
1000 12709 3293 662
767 28086 21734 197
971 8297 17874 583
5 10499 27413 725
936 27616 25250 112
762 20770 16094 191
392 384 14787 345
412 10087 8236 451
858 1763 4640 562
633 20977 2009 350
732 8955 28142 456
878 2737 16952 667
795 5166 19173 105
873 21556 17688 777
152 21264 15965 950
72 4563 2178 211
351 15282 5879 349
81 25335 22505 926
101 23813 4621 334
49 26330 3714 793
32 27843 26553 404
883 17904 7917 582
51 11035 4538 741
421 18475 26513 573
503 12123 6613 526
950 13936 6136 617
541 402 13248 197
955 19941 15838 600
711 3987 3732 178
926 5594 13347 734
333 6329 16240 961
258 26105 11224 680
626 25760 391 810
485 14927 21067 836
145 17377 6476 794
693 20100 19176 564
338 20404 20856 653
87 5263 19869 827
607 25771 3361 828
654 25216 24152 245
842 21631 4953 607
168 27678 7821 822
353 24312 7292 790
528 11397 22396 324
175 27865 29904 796
827 19613 9543 948
814 28786 25556 470
417 27820 24270 511
350 7610 24813 428
652 5517 2996 711
121 25356 22713 446
86 14648 28313 239
52 6260 25587 252
59 24878 18814 190
570 23225 28269 908
343 10338 26827 416
556 13692 27101 745
983 21559 23971 886
978 17103 4404 588
218 15063 12930 667
603 27621 3559 613
858 2013 14516 599
528 1427 16527 413
477 1871 10531 180
1000 11384 25478 462
861 20982 22196 174
522 5190 4913 156
799 22092 7737 386
528 25753 12963 730
75 737 28941 662
985 27602 8481 354
692 1461 24060 278
563 14691 14806 538
993 10775 3874 886
646 25515 4771 408
197 25796 26962 967
231 20076 20251 869
369 19657 7206 316
605 29706 14642 627
523 28648 6953 434
499 1437 22970 962
821 843 22005 261
92 10818 17009 674
687 6640 6642 966
879 25269 24020 412
558 14248 12422 280
319 25514 5950 234
535 17841 10209 622
211 7033 10891 698
364 7383 5448 139
602 29456 13441 184
27 288 17568 964
745 24905 13010 314
365 15303 23461 432
982 9684 14794 508
826 4516 1926 277
695 4388 2129 805
711 26059 21451 676
836 26312 17797 421
826 2214 11970 912
337 15570 26398 406
134 17054 17082 174
368 11115 28761 140
172 18193 18220 216
657 2867 22927 630
375 24720 1857 602
101 7210 15473 590
191 21278 1379 134
867 26880 17260 994
814 9352 13334 423
688 14181 19876 973
329 25860 27820 848
359 4338 690 206
791 11733 17451 502
124 26243 13822 139
963 8315 22647 318
494 2181 5285 572
202 18660 6837 378
149 1889 23223 650
222 25830 27033 265
530 13933 28948 577
13 24320 22692 875
28 5137 15215 111
858 25990 12139 735
177 25351 28761 548
404 28378 10072 255
118 27905 7502 802
894 340 15898 318
474 9424 16493 300
599 7838 24659 488
795 21978 27845 509
827 25652 17629 880
896 20003 13041 442
917 15651 27459 416
715 28387 6939 198
891 1960 20328 240
988 19716 19807 301
190 1938 22946 982
63 15337 14640 831
780 1889 23100 439
762 21027 952 994
972 26030 16328 875
575 38 15729 584
193 466 6012 493
265 13533 14100 684
669 17807 24041 538
379 15083 22223 641
528 29158 20247 182
541 26641 21638 666
148 2512 27768 913
930 22722 5172 138
334 24475 28379 499
157 13599 12701 320
621 20177 15369 280
380 3999 9426 747
147 19259 7366 482
256 11039 10617 692
358 15403 14854 921
702 7639 12796 952
734 23035 1119 863
3 23891 1024 371
935 3623 29860 389
963 26667 25812 330
173 27109 6798 729
448 2428 9636 828
853 14157 28745 550
321 12906 16372 659
434 20055 29462 112
313 28436 16800 125
448 28600 3382 667
511 2410 21219 531
421 2683 21991 594
419 14220 19230 918
303 15271 28238 945
838 6975 25678 327
743 21623 7917 719
96 15408 12880 429
850 12543 22691 968
573 361 17413 228
192 13346 15839 223
501 5962 2048 205
871 17155 22963 345
929 10060 20309 711
891 1024 25936 662
912 27338 21231 292
820 23761 28895 941
710 16392 15617 736
372 22253 570 916
298 20873 13206 454
907 168 27648 397
722 16836 16189 860
352 22612 17651 660
133 25615 12766 841
68 17952 13806 632
363 1129 23844 448
85 17734 18010 932
392 2734 428 995
196 21650 29131 637
90 2917 16642 431
687 6793 4796 395
64 26171 9925 980
981 13483 20064 934
600 11343 2595 969
587 2036 24896 592
722 23330 10820 686
203 10269 7260 673
510 292 17173 164
949 22197 4890 217
328 19364 10801 995
361 8308 5604 841
448 4312 7125 156
815 1386 10179 930
839 11536 25694 467
508 393 17772 493
110 12018 24792 341
667 11410 29878 172
476 9346 2791 584
425 6836 6135 470
436 29224 19072 307
663 13250 29820 245
368 1853 8898 575
985 27227 29143 293
104 2446 7966 981
445 25122 10498 461
948 28272 6682 379
831 4725 7772 725
803 22619 29203 806
935 10465 29511 861
1 6654 13067 557
194 12337 23850 314
637 12650 16613 624
968 23433 25466 289
414 3267 27569 990
121 24597 9685 263
903 24923 27926 143
142 837 8080 734
322 8142 972 208
198 27867 11076 723
601 22834 14785 913
228 18920 15867 739
836 12994 24539 922
875 25956 19795 681
237 2129 24765 904
728 19731 28993 121
882 13401 19517 760
666 18079 18239 554
773 5619 21177 335
203 12353 27602 630
661 18039 22359 428
90 2535 11606 387
248 4678 6022 328
492 1327 17756 633
37 24373 29880 884
481 18703 26712 347
245 29606 28550 715
990 3799 17344 710
293 19650 11921 607
462 9734 17587 635
883 28061 24348 651
154 13374 11645 871
812 2542 3025 766
615 29645 14779 849
478 5859 12454 416
269 24467 15223 611
380 19561 5882 446
554 15193 16539 503
226 4788 11074 356
457 1731 25166 792
159 8621 8932 259
136 7774 23992 168
128 29965 21568 466
191 4396 7746 393
252 22922 10553 350
8 15157 20701 601
358 7586 15739 767
68 26167 12657 158
160 28227 14014 240
268 28295 28102 851
32 23747 8180 687
994 11222 11959 393
753 19597 12592 101
976 6760 24621 357
507 26685 22006 550
304 8574 3824 713
39 24463 837 437
542 26658 9078 403
463 1573 6510 519
401 18456 8919 749
897 23153 21116 588
207 28224 15757 358
973 5713 17025 661
99 23277 11172 511
292 21490 7484 590
918 16185 8281 500
855 18733 8081 955
542 11256 9402 102
749 24790 26553 392
985 955 20779 924
734 10957 11095 372
446 29185 10497 157
875 13089 11194 576
778 1144 8240 872
284 11633 20895 630
359 13177 3497 216
671 12108 2740 147
953 11007 28739 530
441 2687 16318 485
575 19886 12503 600
419 17801 2994 496
981 3197 9096 299
939 2264 24627 872
66 26297 12370 968
86 14736 23359 757
306 15038 217 422
308 1746 15112 157
193 29234 21802 298
907 21612 16812 208
852 24239 27447 129
55 28790 16191 188
401 24120 20411 693
715 1486 16156 394
709 16020 19503 901
600 3368 5288 603
759 3145 25798 299
963 28017 9414 534
902 12166 13230 815
752 18308 17859 908
939 14152 24752 194
991 1654 9136 538
517 28051 23296 615
592 16469 9907 542
653 22350 28348 819
634 1967 15136 120
431 4887 6772 172
184 10182 19937 939
14 25123 12604 176
240 15356 10436 573
645 11108 4455 604
195 14589 28439 824
259 25199 3142 840
331 7244 16940 208
439 25762 19976 348
55 21974 3654 427
652 420 15503 780
431 2243 12395 875
356 28075 3400 832
818 21902 5501 278
312 1428 29321 605
615 17642 9594 316
884 10053 9725 305
677 23290 24745 298
401 20680 5669 426
14 4831 17405 583
254 11721 13368 725
928 23423 14998 772
688 19239 23762 721
82 2303 1798 726
36 317 14051 387
911 20064 9947 785
148 27311 1501 314
993 18162 6611 869
724 5247 2312 501
280 5651 5011 708
448 968 458 571
811 18530 24147 825
61 5198 10917 476
117 17971 8414 852
724 9265 6790 455
929 4068 18415 877
218 5653 4343 511
909 9498 29981 987
72 19357 5331 148
413 16234 20834 986
534 7473 12417 183
397 28003 18296 343
255 4120 8917 779
406 13184 1557 344
891 29976 11350 305
441 17800 11288 994
726 6864 3299 641
475 17490 18198 793
273 16361 18099 931
240 2022 23120 809
24 15494 3148 129
885 6429 21156 680
762 18127 13182 968
220 16554 6837 728
825 23945 27327 581
801 5300 2683 218
561 23073 12108 901
2 7800 29098 122
618 13025 14149 187
242 1854 23717 874
788 13355 24810 861
691 8360 16574 858
247 2515 21370 977
542 8047 9661 142
846 14103 16605 173
855 22199 10327 917
573 27472 28483 647
940 20166 23849 580
267 23079 16492 610
456 25957 517 279
887 21054 11597 894
110 28429 19547 737
838 307 18311 370
35 14692 19808 736
811 17607 22297 332
554 10102 23605 893
773 6820 27649 643
767 19580 8584 322
588 16654 22043 552
654 15049 6091 232
451 14491 10014 816
996 18837 19059 565
594 25159 2166 450
912 27147 7849 318
818 24685 20866 244
749 18584 24681 698
980 6337 16965 397
337 9985 18992 970
656 10788 13242 592
763 4652 1836 162
688 1356 3132 271
917 12952 1049 473
321 23734 24330 623
499 27069 7316 151
927 16831 5521 606
233 10264 25626 342
355 24241 2546 142
148 17117 2742 178
825 14188 17705 353
200 8093 7596 377
376 8520 5204 374
104 8875 24419 247
838 24030 24174 497
513 7070 10324 119
777 6867 21709 647
957 9924 16385 353
836 17134 27594 607
726 16035 20175 101
954 18102 4869 256
541 10057 244 645
902 27378 26435 632
481 24414 5112 535
366 23257 10310 861
126 25674 14249 268
948 27961 20167 795
492 22673 14395 985
97 13445 21857 652
132 14989 27985 701
76 11034 16987 542
682 19557 7075 570
750 2047 15282 931
396 6876 24908 949
615 5343 25746 779
434 14458 22212 383
593 26882 2351 841
748 18725 11262 606
810 4823 22746 741
691 13491 22265 660
648 23792 24645 248
192 29043 27539 862
37 3159 28246 555
244 23470 18772 504
327 5836 2177 370
866 23074 20639 626
633 11027 29740 573
578 27675 28555 331
71 15226 25154 658
987 20615 12334 333
460 17375 22944 257
961 23055 3622 440
824 24640 27650 499
526 27041 10172 554
644 27158 14349 813
45 17221 26679 336
134 3116 7417 550
774 1472 23823 718
297 23585 25430 606
292 16610 24526 741
33 25115 13211 380
626 23911 12608 313
509 19621 26337 674
253 23554 5998 204
760 12386 4477 791
659 28533 11603 254
94 10307 16596 454
986 11385 2458 507
829 11371 4685 216
932 29806 7286 359
303 9793 16879 700
960 22555 15599 318
523 28719 16722 270
553 14533 24687 1000
177 1999 22511 971
622 515 27901 607
296 8802 27052 586
216 14580 1359 517
2 27519 9628 232
490 7961 21947 569
920 28012 8596 369
705 8825 18507 182
192 5881 23114 334
495 7166 14395 357
499 6142 5638 783
309 20715 28996 426
512 12310 28002 492
429 1353 11352 761
935 9701 16286 754
345 14991 23149 359
255 26536 14424 699
953 22101 19812 470
945 5451 19879 920
759 2253 25317 515
462 18622 9561 201
190 12242 13957 537
949 23476 16816 838
516 22293 29479 718
11 5123 234 742
604 9315 27835 585
484 10900 6627 169
735 20242 21155 634
658 21548 28890 670
527 7847 25936 437
252 8475 12104 983
534 17768 8630 693
234 2896 3834 650
633 4189 6715 764
961 14468 7878 728
65 24047 23512 244
760 14723 24600 523
169 19638 5255 335
439 29420 6121 147
231 16324 20133 882
244 10465 1951 628
813 228 391 734
388 27125 16512 793
677 27452 17163 603
851 4436 15684 617
393 19371 13600 447
117 12184 24709 736
109 6700 986 481
563 704 2003 230
260 8375 19122 168
379 13999 28160 425
139 3407 207 971
422 1342 1276 676
155 17815 22222 267
535 29623 3759 954
286 15995 24411 156
5 26200 8819 537
956 28616 22221 680
448 3686 14696 626
392 5586 766 165
553 17814 13557 386
844 19059 4144 439
184 24378 23705 129
191 4821 10923 855
400 15547 8941 221
372 13355 4201 947
193 17819 3942 777
417 3936 23485 232
321 21321 23608 183
408 516 2441 880
118 27944 11739 764
38 20660 7957 917
434 18789 29723 988
849 26805 2732 421
18 3393 18232 735
413 11435 24670 131
860 8085 5263 429
601 26620 12635 138
462 27070 15513 293
518 12851 28633 830
359 1303 28849 439
669 9559 19418 934
164 1686 3106 934
59 19928 1231 931
768 270 21095 531
456 14720 11360 412
833 14269 17893 303
212 10702 25970 321
243 28570 28461 358
897 20911 11578 963
61 4412 5683 531
702 2471 22342 572
314 6821 18191 851
784 28122 21186 329
605 11616 5935 832
970 29816 28868 961
135 6212 21099 220
648 9031 9167 717
546 17060 7017 175
836 9600 15929 133
603 24501 14173 823
695 8386 20757 312
576 11299 10317 964
341 24545 20185 769
419 26475 29834 351
356 28066 25910 482
783 22902 12871 250
334 4970 2178 360
583 3288 26432 172
159 16278 24275 644
47 19734 15849 326
244 8739 25055 742
967 18662 17532 785
422 27674 14328 473
620 23556 24743 309
699 6953 20988 727
790 10633 23097 547
643 25300 16655 165
583 22041 18223 390
46 12178 1552 362
467 5364 14727 795
882 26240 9114 722
391 18501 24800 964
722 441 14624 876
309 20863 18354 445
228 29303 19762 439
207 9162 4183 746
167 10253 17433 973
855 11985 10801 533
969 28088 3490 505
451 711 6973 676
789 12525 19989 193
965 23450 623 427
446 2544 28720 801
567 23005 373 395
332 6086 447 547
581 27538 12191 176
73 27901 5902 998
958 26876 25010 313
39 13746 26721 813
390 29356 15455 638
393 24836 8946 999
54 9311 22986 856
23 15385 4699 248
815 1955 1729 918
463 21240 18372 299
929 10420 1844 117
355 18320 28093 966
10 20745 27734 912
232 10754 16992 525
922 10638 28110 623
629 25296 23675 812
546 16796 1527 148
274 10008 26069 857
70 19202 11981 253
8 24776 22701 645
824 25370 17412 747
925 26055 11667 103
703 26198 7628 839
65 12331 13473 818
552 13610 23878 596
420 13177 16745 516
44 6841 21229 748
502 29052 177 803
291 4325 29287 788
138 13567 25980 447
192 12840 29689 975
705 18608 29057 510
215 7608 1023 500
285 20987 10561 837
117 29328 8531 815
931 22618 21169 443
102 6573 2948 503
151 17239 18121 525
864 17694 25114 355
950 16568 20795 499
405 24517 13580 869
826 28689 6249 112
124 12942 2762 229
602 18406 13120 199
581 9703 13602 605
723 14083 19661 268
320 17938 24641 770
648 26270 18955 128
696 29053 25694 385
616 23521 12240 688
560 8739 15642 161
282 8513 29658 225
293 24214 2909 973
580 24015 4952 959
204 18899 24338 624
545 12851 24063 161
592 19592 14306 405
808 14560 16495 248
354 24719 359 578
970 19635 19664 546
7 4262 3148 597
272 9586 26230 149
444 4827 20501 532
309 15341 20189 604
631 27252 7712 332
673 5439 22031 796
958 6378 26867 213
888 28927 24701 540
413 22247 3603 165
99 25036 16392 416
24 5543 23617 468
175 5332 13607 292
561 10806 21115 414
910 29585 1590 746
195 5867 20843 104
891 29083 21221 327
853 28303 17998 685
532 11951 14370 512
443 15555 8977 552
893 29618 3700 127
269 13453 4956 872
473 18178 10961 170
678 23668 16966 852
225 4568 13742 841
645 7077 13534 782
111 15176 18250 259
951 25553 25336 124
491 20843 19162 167
665 13759 26601 996
387 6138 2901 436
576 29064 13250 384
391 28801 8837 310
411 11035 16325 652
832 23873 27458 808
611 7134 17304 692
594 18101 18784 699
362 13259 17151 674
535 16748 372 751
865 4462 28344 856
350 4607 8698 446
450 6760 1457 343
847 2633 21517 955
58 11809 28537 460
67 15652 28877 393
674 20428 10576 771
264 15208 15215 509
188 29128 11564 990
125 5537 25917 883
633 26926 9777 978
390 831 21809 831
417 28448 13261 124
721 28835 16796 399
368 23394 5342 560
182 29631 25767 840
608 5369 19462 274
272 14084 26873 295
534 13904 27086 252
243 29923 14629 819
326 26011 1048 876
531 21399 20732 620
301 4898 17959 210
422 3482 7813 497
183 10029 27328 189
9 24807 6717 106
391 3542 27902 772
575 985 26508 548
358 27901 24366 186
352 24240 22096 443
