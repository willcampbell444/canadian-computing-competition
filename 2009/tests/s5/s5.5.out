414374
11672
